third_party/chromium/media/base/audio_codecs.h

```diff
@@ -33,13 +33,20 @@ enum class AudioCodec {
   kALAC = 15,
   kAC3 = 16,
   kMpegHAudio = 17,
+#if defined (STARBOARD)
+  kIAMF = 18,
```

Comment:
Good catch, I'll see if I can check with Chromium

Reply:
Chromium requires the enum values to be fixed to avoid affect telemetry info.  It's great if we can align with them, but if we cannot find out the exact value they are going to use, it's ok to use a tentative value, as long as this doesn't pollute our telemetry, where Thor should have more info.